packages/android-app/package.json

```diff
@@ -4,17 +4,19 @@
   "private": true,
   "license": "BSD-3-Clause",
   "scripts": {
-    "bundle:deps": "cd android && ./gradlew generateReactArchives && cd ..",
-    "bundle:js": "haul bundle --platform android --dev false --entry-file js/index.android.js --bundle-output assets/js/index.android.bundle --assets-dest assets/res/",
-    "build": "yarn bundle:deps && yarn bundle:js",
-    "fmt": "prettier --write 'src/**/*.*'",
+    "build:deps": "cd android && ./gradlew generateReactArchives && cd ..",
+    "build:js": "haul bundle --platform android --dev false --entry-file js/index.android.js --bundle-output assets/js/index.android.bundle --assets-dest assets/res/",
```

Comment:
platform android? I'd say `--platform all`

Reply:
Planning to do iOS builds on a separate package called `ios-app`
Any shareable code should go into the appropriate packages like article or author-profile, and we'll use android-app and ios-app packages for platform specifics and native bridges